tools/nitrogfx/main.c

```diff
@@ -47,6 +47,28 @@ void ConvertGbaToPng(char *inputPath, char *outputPath, struct GbaToPngOptions *
 
 void ConvertNtrToPng(char *inputPath, char *outputPath, struct NtrToPngOptions *options)
 {
+    // handle empty files if possible
+    FILE *fp = fopen(inputPath, "rb");
+
+	if (options->handleEmpty)
```

Comment:
Should be 4-spaces, instead of a tab

```suggestion
    if (options->handleEmpty)